scene/2d/camera_2d.cpp

```diff
@@ -513,6 +521,39 @@ float Camera2D::get_follow_smoothing() const {
 	return smoothing;
 }
 
+void Camera2D::set_enable_snapping(bool p_enabled) {
```

Comment:
`set_snapping(bool p_enabled)` would be more idiomatic in the Godot code base (not 100% consistent everywhere, but it's the most common convention).

Reply:
`is_snapping_enabled()` for the getter is already fine btw.

Reply:
And well, `set_enable_follow_smoothing` in this very class contradicts my "convention", so... do as you see best fit ;)